plugins/polio/js/src/domains/VaccineModule/StockManagement/Details/Table/useVaccineStockManagementDetailsColumns.tsx

```diff
@@ -125,3 +125,52 @@ export const useVaccineStockManagementDetailsColumnsUnusable = (): Column[] => {
         return columns;
     }, [formatMessage]);
 };
+export const useVaccineStockManagementDetailsColumnsEarmarked =
+    (): Column[] => {
+        const { formatMessage } = useSafeIntl();
+        return useMemo(() => {
+            const columns = [
+                {
+                    Header: formatMessage(MESSAGES.date),
+                    accessor: 'date',
+                    id: 'date',
+                    Cell: DateCell,
+                },
+                {
+                    Header: formatMessage(MESSAGES.action),
+                    accessor: 'action',
+                    id: 'action',
+                    Cell: settings => {
+                        const { action } = settings.row.original;
+                        return MESSAGES[action]
+                            ? formatMessage(MESSAGES[action])
+                            : action;
+                    },
+                },
+                {
+                    Header: formatMessage(MESSAGES.vials_in),
+                    accessor: 'vials_in',
+                    Cell: settings => {
+                        const { vials_in } = settings.row.original;
+                        if (!vials_in) {
+                            return <span>{textPlaceholder}</span>;
+                        }
+                        return <NumberCell value={vials_in} />;
```

Comment:
is this required ? NumberInput => FormattedNumber => NumericFormat is already handling placeholder normally